clustering/infinispan/extension/src/test/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanTransformersTestCase.java

```diff
@@ -97,6 +97,8 @@ private static InfinispanModel getModelVersion(ModelTestControllerVersion contro
                 return InfinispanModel.VERSION_4_0_0;
             case EAP_7_1_0:
                 return InfinispanModel.VERSION_5_0_0;
+            case EAP_7_2_0_TEMP:
+                return InfinispanModel.VERSION_8_0_0;
```

Comment:
@pferraro Now that the core upgrade has been merged this should be changed to just EAP_7_2_0